### ISSUE TITLE
Cannot build project with Parchment in pods

### ISSUE THREAD

**Issue by andrii-lisovskiy** [rechsteiner/Parchment#599]
Hello, I've just installed xCode 13 and now cannot build app, errors in Parchment.

My stack is:
xCode 13.0 (13A233)
CocoaPods 1.11.2
Parchment (3.0.1)

Tried pod repo update, pod deintegrate and install again, cleaning derived data - nothing helps.

<img width="237" alt="Screenshot 2021-09-23 at 12 37 16" src="https://user-images.githubusercontent.com/69899396/134485560-866b02cc-0f31-450b-8c8a-a68c6add0a9f.png">

**Comment by rechsteiner** [comment]
Hi @andrii-lisovskiy! This is due to a bug in Xcode 13: https://github.com/rechsteiner/Parchment/issues/594#issuecomment-925155470. You can target the `xcode-13` branch for now. If this isn't fixed in the official release, I will look into either dropping iOS <11 or splitting the SwiftUI code into a separate package.

**Comment by andrii-lisovskiy** [comment]
Thanks for quick response, waiting for fix. 
BTW, looks like with SPM everything works smoothly.

**Comment by rechsteiner** [comment]
Yeah, installing and building works fine, but you get the same issue when archiving the app. Did the `xcode-13 ` branch work for you?

**Comment by andrii-lisovskiy** [comment]
Tried
`pod 'Parchment', :git => 'https://github.com/rechsteiner/Parchment.git', :branch => 'xcode-13'`

And still I have errors, cannot build.

**Comment by rechsteiner** [comment]
Hmm, okay. This might be a different issue then. Need to look more into it

**Comment by marekhac** [comment]
Yes, I confirm that the fix from `xcode-13` doesn't work :(. I hope you will find the solution soon. Thanks in advance.

**Comment by rechsteiner** [comment]
Feel free to fork it and test out some stuff ourselves though. I’m a bit busy these days, but happy to merge any fixes

**Comment by marekhac** [comment]
The only workaround that I found is to use version `2.2.0` of Parchment.

**Comment by Mandasy** [comment]
I also encountered the same problem as @andrii-lisovskiy, is there any solution?

**Comment by tgdong2296** [comment]
I have same problem too. Hope you fix that soon. 
Thank you !

**Comment by dlackty** [comment]
One way to workaround this issue is to raise the deployment target of this pod to 11.0:

Add this to your `Podfile`:

```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      # https://github.com/CocoaPods/CocoaPods/issues/9884
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '11.0'
    end
  end
```

**Comment by DAEUN28** [comment]
In my case, this problem occurred when Xcode beta version(13) and Xcode release version(13) were installed together.
It may be because `xcode-select` with beta version. 
So you can solve the problem by deleting the Xcode beta version or by run `xcode-select` with the xcode version you are currently using.

**Comment by clbemre** [comment]
> One way to workaround this issue is to raise the deployment target of this pod to 11.0:
> 
> Add this to your `Podfile`:
> 
> ```ruby
> post_install do |installer|
>   installer.pods_project.targets.each do |target|
>     target.build_configurations.each do |config|
>       # https://github.com/CocoaPods/CocoaPods/issues/9884
>       config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '11.0'
>     end
>   end
> end
> ```

Hello, this really worked. Thank you.
I have a question for you :)
I made the deployment version 12.0. What is the difference between making 11.0 12.0 or 13.0?
Deployment target of one of my projects is 12.0 and the other is 13.0

**Comment by clbemre** [comment]
> > One way to workaround this issue is to raise the deployment target of this pod to 11.0:
> > Add this to your `Podfile`:
> > ```ruby
> > post_install do |installer|
> >   installer.pods_project.targets.each do |target|
> >     target.build_configurations.each do |config|
> >       # https://github.com/CocoaPods/CocoaPods/issues/9884
> >       config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '11.0'
> >     end
> >   end
> > end
> > ```
> 
> It's better to set deployment target for Parchment only, like [here](https://github.com/CocoaPods/CocoaPods/issues/9884#issuecomment-696228403).

actually i was getting errors on most pods but after doing that all the errors are gone. :)
